src/EditorFeatures/CSharpTest/Diagnostics/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -7154,5 +7154,27 @@ public Foo(String foo)
     public string String { get; private set; }
 }");
         }
+
+        [WorkItem(18275, "https://github.com/dotnet/roslyn/issues/18275")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddImport)]
+        public async Task TestContextualKeyword1()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"
+namespace N
+{
+    class nameof
```

Comment:
I don't see value in that.  I like tests to test concepts and important codebranches.  I don't want tests to just exhaustively test all codepaths that could be hit.  Importantly, i find anti-value in tests that are just verifying trivial checks, or APIs that are tested through other means.  So, for example, SyntaxFacts.IsContextualKeyword is well tested through other means.  As such, i don't want just duplicate that sort of coverage in tests here.

This was an explicit design choice we made around tests when we moved to Roslyn.  Our previous native system had excessively redundant tests that were just revalidating the same lower layers unnecessarily.  So we moved to a model where tests should try to validate the parts of code that their feature is responsible for.